fe/fe-core/src/main/java/com/starrocks/credential/hdfs/HDFSCloudConfiguration.java

```diff
@@ -14,38 +14,95 @@
 
 package com.starrocks.credential.hdfs;
 
-import autovalue.shaded.com.google.common.common.base.Preconditions;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
 import com.staros.proto.FileStoreInfo;
+import com.starrocks.StarRocksFE;
 import com.starrocks.credential.CloudConfiguration;
 import com.starrocks.credential.CloudType;
 import com.starrocks.thrift.TCloudConfiguration;
+import com.starrocks.thrift.TCloudType;
 import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.starrocks.credential.CloudConfigurationConstants.HDFS_CONFIG_RESOURCES;
+import static com.starrocks.credential.CloudConfigurationConstants.HDFS_CONFIG_RESOURCES_LOADED;
+import static com.starrocks.credential.CloudConfigurationConstants.HDFS_FS_CREDENTIAL_KEY;
+import static com.starrocks.credential.CloudConfigurationConstants.HDFS_RUNTIME_JARS;
 
 public class HDFSCloudConfiguration implements CloudConfiguration {
+    private static final Logger LOG = LogManager.getLogger(HDFSCloudConfiguration.class);
+
     private final HDFSCloudCredential hdfsCloudCredential;
+    private String configResources;
+    private String runtimeJars;
 
     public HDFSCloudConfiguration(HDFSCloudCredential hdfsCloudCredential) {
         Preconditions.checkNotNull(hdfsCloudCredential);
         this.hdfsCloudCredential = hdfsCloudCredential;
     }
 
+    public void setConfigResources(String configResources) {
+        this.configResources = configResources;
+    }
+
+    public void setRuntimeJars(String runtimeJars) {
+        this.runtimeJars = runtimeJars;
+    }
+
     public HDFSCloudCredential getHdfsCloudCredential() {
         return hdfsCloudCredential;
     }
 
+    public void addConfigResourcesToConfiguration(String configResources, Configuration conf) {
+        if (Strings.isNullOrEmpty(configResources)) {
+            return;
+        }
+        String[] parts = configResources.split(",");
```

Comment:
Using a constant to represent `,`, that will be more clear. Like `String CONFIG_RESOURCES_SEPERATOR = '.'`?

Reply:
OK, will fix that.